src/core/eme/eme_manager.ts

```diff
@@ -112,8 +113,12 @@ export default function EMEManager(
   const handledInitData = new InitDataStore();
 
   const mediaKeysInfos$ = // store the mediaKeys when ready
-    initMediaKeys(mediaElement, keySystemsConfigs, attachedMediaKeysInfos)
-      .pipe(take(1), shareReplay());
+    mediaSourceReady$.pipe(
```

Comment:
Wouldn't that be `Init`'s role to only "launch" the EMEManager once the MediaSource is ready?

Reply:
Plus it removes the `mediaSourceReady` arguments from the EMEManager which is not that useful here.